runtime/development/src/lib.rs

```diff
@@ -879,6 +880,7 @@ impl Contains<CurrencyId> for PoolCurrency {
 			| CurrencyId::KSM
 			| CurrencyId::Permissioned(_) => false,
 			CurrencyId::AUSD | CurrencyId::KUSD => true,
+			CurrencyId::ForeignAsset(_) => todo!("nuno: it would be interesting to decide this based on some metadata field stored in AssetRegistry?")
```

Comment:
`todo`: shall we add a `pool_currency: bool` field to `CustomMetadata` and resolve this based on that? Other ideas? Maybe a `token_type` instead, that would be either `Stable | Native | Other |  Etc`?